source/detail/parser/x86_64.cpp

```diff
@@ -31,6 +33,110 @@ namespace smeagle::x86_64 {
     MEMORY        // Types that will be passed and returned in memory via the stack
   };
 
+  // A FramebaseAllocator keeps track of framebase index
+  class FramebaseAllocator {
+  private:
+    int nextMultipleEight(int number);
+    int framebase = 8;
+    void updateFramebaseFromType(st::Type *paramType);
+
+  public:
+    std::string nextFramebaseFromType(st::Type *paramType);
+  };
+
+  // Get a framebase for a variable based on stack location and type
+  void FramebaseAllocator::updateFramebaseFromType(st::Type *paramType) {
```

Comment:
We should leave a comment here about the required frame alignment.

Reply:
8 _byte_ aligned.

Reply:
Fixed.